docs/source/certs_management.md

```diff
@@ -0,0 +1,382 @@
+
+### Organization CA Certificates
+
+Organization CA Root Certificates and Organization CA Admin Certificates are self-signed certificates if creating a new Certificate Authority (CA), or provided by an external CA.
+
+#### Organization CA Root Certificate
+
+**Description**: Public Certificate that permits verification of all certificates issued by the Organization CA.
+
+**Location**: Stored on disk in the Organization CA directory (ca-cert.pem), and copied into the channel configuration to verify identifies for the organization.
+
+**Impact if expired**: A new Organization CA Root Certificate must be issued. Organization CA Root Certificates are valid for 15 years.
+
+
+#### Organization CA Admin Certificate
+
+**Description**: Certificate used when making admin requests to the Organization CA.
+
+**Location**: Dependent on implementation:
+
+**Note**: Each identity has a local **msp** directory structure which contains its certificate in the **signcerts** directory and its private key in the **keystore** directory. For details on the **msp** directory, refer to [MSP Structure](https://hyperledger-fabric.readthedocs.io/en/latest/membership/membership.html#msp-structure).
+
+<pre>
+msp
+ ├── IssuerPublicKey
+ ├── IssuerRevocationPublicKey
+ ├── cacerts
+ │   └── localhost-7053.pem
+ ├── keystore
+ │   └── key.pem
+ ├── signcerts
+ │   └── cert.pem
+ └── user
+</pre>
+
+**Impact if expired**: The Organization Administrator cannot register or enroll new identities with the CA, but transaction traffic does not stop.
+
+[Reference - Enroll Orderer Org’s CA Admin](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer-org-s-ca-admin)
+
+
+### TLS CA Certificates
+
+TLS CA Root Certificates and TLS CA Admin Certificates are self-signed certificates if creating a new Certificate Authority (CA), or provided by an external CA.
+
+
+#### TLS CA Root Certificate
+
+**Description**: Certificate used when making admin requests to the TLS CA.
+
+**Location**: Stored on disk in the TLS CA directory (ca-cert.pem), and copied into the channel configuration to verify identifies for the organization.
+
+**Impact if expired**: A new TLS CA Root Certificate must be issued. TLS CA Root Certificates are valid for 15 years.
+
+
+#### TLS CA Admin Certificate
+
+**Description**: Certificate used for admin requests to the TLS CA.
+
+**Location**: Dependent on implementation:
+
+<pre>
+msp
+ ├── IssuerPublicKey
+ ├── IssuerRevocationPublicKey
+ ├── cacerts
+ │   └── localhost-7053.pem
+ ├── keystore
+ │   └── key.pem
+ ├── signcerts
+ │   └── cert.pem
+ └── user
+</pre>
+
+**Impact if expired**: The Fabric Administrator will no longer be able to issue or renew TLS certificates for nodes in the network.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-tls-ca-s-admin)
+
+
+### Peer Certificates
+
+Peer Enrollment Certificates and Peer TLS Certificates are issued for each organization.
```

Comment:
```suggestion
Peer Enrollment Certificates and Peer TLS Certificates are issued for each peer of each organization.
```

Reply:
A Peer Enrollment Certificate and a Peer TLS Certificate are issued for each peer in an organization.